beacon-chain/rpc/prysm/v1alpha1/validator/proposer_capella.go

```diff
@@ -27,3 +34,111 @@ func (vs *Server) setBlsToExecData(blk interfaces.BeaconBlock, headState state.B
 		}
 	}
 }
+
+func (vs *Server) unblindBuilderCapellaBlock(ctx context.Context, b interfaces.SignedBeaconBlock) (interfaces.SignedBeaconBlock, error) {
```

Comment:
Rename to `unblindBuilderBlockCapella`? Same argument as in the mock.

Reply:
Also I don't particularly like that we just return `nil, err` several times without wrapping the error in a meaningful message.